scripts/run_lighthouse_tests.py

```diff
@@ -96,12 +141,14 @@ def export_url(url):
         return
 
 
-def run_lighthouse_checks():
+def run_lighthouse_checks(is_performance_mode):
     """Runs the lighthouse checks through the .lighthouserc.js config."""
-
-    node_path = os.path.join(common.NODE_PATH, 'bin', 'node')
     lhci_path = os.path.join('node_modules', '@lhci', 'cli', 'src', 'cli.js')
-    bash_command = [node_path, lhci_path, 'autorun']
+    if is_performance_mode:
```

Comment:
Specify the path explicitly in both cases for parity.

This also means that you can reduce the branching. Instead, do something like:

```
LIGHTHOUSE_CONFIG_PATHS = {
    LIGHTHOUSE_MODE_PERFORMANCE: ...,
    LIGHTHOUSE_MODE_ACCESSIBILITY: ...,
}
```

and then you can have just a single assignment for bash_command, without the if/else.

Reply:
Done